models/yang/sonic/sonic-vrf.yang

```diff
@@ -24,9 +33,12 @@ module sonic-vrf {
                 key "vrf_name";
 
                 leaf vrf_name {
-                    type string;
-                    description
-                        "Vrf name for SONiC, must start with Vrf.";
+                    type string {
+                        pattern "Vrf[a-zA-Z0-9_-]*" {
```

Comment:
The "*" at the end would allow a vrf with name "Vrf". Do we want to allow that?

Reply:
changed to + to enforce no "Vrf" can be used alone